src/components/SideNav/SideNav.jsx

```diff
@@ -14,155 +14,6 @@ import { COLORS, PADDING } from '../../styles/styles';
-//       background: transparent;
-
-//       &.bx--side-nav__toggle {
-//         background: ${COLORS.navToggle};
-
-//         :hover {
-//           background: ${COLORS.gray80};
-//         }
-//       }
-//     }
-
-//     button.bx--side-nav__link {
-//       border: none;
-//     }
-
-//     .bx--side-nav__link > .bx--side-nav__link-text {
-//       white-space: nowrap;
-//       overflow: hidden;
-//       text-overflow: ellipsis;
-//       color: ${COLORS.gray10};
-//       font-size: 0.875rem;
-//       letter-spacing: 0.1px;
-//       line-height: 1.25rem;
-//       -webkit-user-select: none;
-//       -moz-user-select: none;
-//       -ms-user-select: none;
-//       user-select: none;
-//     }
-
-//     .bx--side-nav__link:focus {
-//       outline: 4px solid ${COLORS.blue60};
-//       outline-offset: -4px;
-//     }
-//   }
-// `;
-// const sStyledSideNavLink = styled(SideNavLink)`
-//   &&& {
-//     position: relative;
-//     display: flex;
-//     align-items: center;
-//     text-decoration: none;
-//     min-height: 3rem;
-//     padding-right: 1rem;
-//     font-weight: 400;
-
-//     &.bx--side-nav__link--current {
-//       background-color: ${COLORS.gray70};
-//     }
-
-//     :focus {
-//       outline: 4px solid ${COLORS.blue60};
-//       outline-offset: -4px;
-//     }
-
-//     :hover {
-//       background-color: ${COLORS.gray80};
-//     }
-
-//     > .bx--side-nav__link-text {
-//       white-space: nowrap;
-//       overflow: hidden;
-//       text-overflow: ellipsis;
-//       color: ${COLORS.gray10};
-//       font-size: 0.875rem;
-//       letter-spacing: 0.1px;
-//       line-height: 1.25rem;
-//       -webkit-user-select: none;
-//       -moz-user-select: none;
-//       -ms-user-select: none;
-//       user-select: none;
-//     }
-
-//     > .bx--side-nav__icon {
-//       margin-right: 0.5rem;
-//     }
-//   }
-// `;
-
-// const StyledSideNavMenu = styled(SideNavMenu)`
-//   &&& {
-//     background-color: ${COLORS.gray100};
-
-//     :not(.bx--side-nav__item--active):hover {
-//       background-color: ${COLORS.gray100};
-//     }
-
-//     > button:hover {
-//       background-color: ${COLORS.gray80};
-//     }
-
-//     :hover {
-//       background-color: ${COLORS.gray80};
-//     }
-
-//     [aria-expanded='true'] {
-//       background-color: ${COLORS.gray80};
-
-//       :hover {
-//         background-color: ${COLORS.gray80hover};
-//       }
-
-//       + ul {
-//         background-color: ${COLORS.gray80};
-//       }
-
-//       + ul .bx--side-nav__link:hover {
-//         background-color: ${COLORS.gray80hover};
-//       }
-//     }
-//     /* We have to apply these styles when we switch out a tag for something else */
-//     .bx--side-nav__link--current::before {
-//       content: '';
-//       position: absolute;
-//       top: 0;
-//       bottom: 0;
-//       left: 0;
-//       width: 4px;
-//       background-color: ${COLORS.blue60};
-//     }
-//   }
-// `;
-
-// new styles
```

Comment:
Can we delete all this commented-out styling?

Reply:
This is deleted. the pink is showing the deletion of this.

Reply:
🤦‍♂ clearly I am out of it, this _is_ deleted